src/libexpr/attr-path.cc

```diff
@@ -57,18 +57,18 @@ std::pair<Value *, PosIdx> findAlongAttrPath(EvalState & state, const std::strin
         Value * vNew = state.allocValue();
         state.autoCallFunction(autoArgs, *v, *vNew);
         v = vNew;
-        state.forceValue(*v, noPos);
+        state.forceValue(*v, pos);
```

Comment:
Are you sure? FWIW it came from https://github.com/NixOS/nix/commit/c1ca4f0accdb96295b27dadacd20b3db745e1d2d